pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -320,6 +351,10 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			if err != nil {
 				return err
 			}
+
+			if mir.Location == "" {
+				return &InvalidRegistries{s: "invalid condition: mirror location is unset"}
+			}
 		}
 		regMap[reg.Location] = append(regMap[reg.Location], reg)
 	}
```

Comment:
What happens in the loop below?

AFAICS with a `{ prefix = *.example; location = ""; insecure = true }; { prefix = *.other; location = ""; blocked = true }` the loop will complain about conflicting insecure/blocked settings.

I think `regMap` should use `reg.Prefix` if `reg.Location` is unset.

Reply:
I'll add this now

Reply:
@mtrmac PTAL ^ .. hope that looks ok.

Reply:
err likely not :\

Reply:
- `		regMap[reg.Location] = append(regMap[reg.Prefix], reg)` should use `reg.Prefix in both cases
- The loop below looking up data in `regMap` needs to consider `reg.Prefix` as well if `reg.Location` is empty.

Reply:
ah, ack, thanks!

Reply:
done, PTAL. Other comments added as FIXMEs. Thanks.